certora/specs/Reverts.spec

```diff
@@ -4,21 +4,6 @@ import "ConsistentInstantiation.spec";
 
 methods {
     function _.price() external => mockPrice() expect uint256;
-
-    function MORPHO.market_(PreLiquidation.Id) external
-        returns (PreLiquidation.Market memory) envfree;
-    function MORPHO.position_(PreLiquidation.Id, address) external
-        returns (PreLiquidation.Position memory) envfree;
-
-    function MathLib.mulDivDown(uint256 a, uint256 b, uint256 c) internal
-        returns uint256 => summaryMulDivDown(a,b,c);
-    function MathLib.mulDivUp(uint256 a, uint256 b, uint256 c) internal
-        returns uint256 => summaryMulDivUp(a,b,c);
```

Comment:
Note that this was not sound: it introduced some `require_uint256`. Luckily, it wasn't needed

Reply:
I'm not sure I understand where computations were unsound.

Reply:
> Do you say so because mulDivUp is being summarized?

Yes exactly.

> I'm not sure I understand where computations were unsound.

It would be unsound in case where we try to prove things about the revert path. For example, consider the following function:
```solidity
function myFunction(uint256 a, uint256 b, uint256 c) external {
   storageVariable = mulDivUp(a, b, c);
}
```
Then with the summarization of `mulDivUp` we would be able to prove that this function never reverts

Reply:
Ok, I see.